src/org/isf/lab/test/Tests.java

```diff
@@ -240,6 +239,30 @@ public void testIoGetLaboratory()
 		
 		return;
 	}
+
+
+	@Test
+	public void testIoGetLaboratoryWithoutDescription()
+	{
+		try	{
```

Comment:
I noted you have switched styles using "{" in same line or in the next line. Personally, I like more opening in the same line, but I'm not sure if here is any specific style rule already adopted.

Reply:
I saw many different approaches within codebase. In other modules { is in the same line so I adopted it also here (and it's java standard)

Reply:
The actual code has indeed this problem of different styles, but in this case it doesn't bring much confusion in diff, not more than indentation (which sure is worst).

Nevertheless, with https://github.com/informatici/openhospital-core/pull/50 the author "added a .editorconfig file, so all editorconfig-aware editors will use tabs from now on."